roles/v2/pool/pool-config.toml

```diff
@@ -5,7 +5,7 @@ listen_jn_address = "127.0.0.1:34264"
 authority_public_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL"
 authority_secret_key = "2Z1FZug7mZNyM63ggkm37r4oKQ29khLjAvEx43rGkFN47RcJ2t"
 cert_validity_sec = 3600
-test_only_listen_adress_plain =  "0.0.0.0:34250"
```

Comment:
This can't change to `34255` because [the tproxy is already listening on this port](https://github.com/stratum-mining/stratum/blob/main/roles/translator/proxy-config.toml#L13).

Reply:
I'd rather you changed the miner to connect on port 34250